Asg0.7-2.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+	"sync"
+)
+
+func main() {
+	originalString := "test123"
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	go func() {
+		defer wg.Done()
+
+		runes := []rune(originalString)
+
+		for currentIndex := 0; currentIndex < len(runes)/2; currentIndex++ {
+			oppositeIndex := len(runes) - 1 - currentIndex
```

Comment:
Can you reduce the number of len() calls to optimise this code?

Reply:
okay, I'll try

Reply:
Nooo.. This hasn't optimised but increased memory footprint.
try to remove len() call that is inside of for loop

Reply:
Superb